www/include/views/graphs/generateGraphs/generateImage.php

```diff
@@ -95,8 +95,6 @@
     } else {
         die('Invalid token');
     }
-} else {
-    throw new \Exception('Username and token query strings must be set.');
```

Comment:
What's wrong with this condition ?

Reply:
@TamazC When you click on "generate image" it doesn't send token nor the username in the query strings so the generate image won't work, also i did add this else before when i did sanitize but i don't remember why.